crawler/pr0gramm.py

```diff
@@ -34,6 +35,7 @@ def _restart_at_front(self):
     def __init__(self, uri, site):
         self.__site = site
         self.__uri = self.__class__.__build_uri(uri)
+        self.__filterNextPage = re.compile('^/static/' + self.__site + '/[\d]+')
```

Comment:
"Next Page" Link looks like this: <a href="/static/new/1164005" style="float: right;">Next Page &raquo;</a>